Lib/test/test_importlib/test_lazy.py

```diff
@@ -48,8 +51,10 @@ def find_spec(self, name, path, target=None):
         return util.spec_from_loader(name, util.LazyLoader(self))
 
     def exec_module(self, module):
+        time.sleep(0.01)  # Simulate a slow load.
```

Comment:
Is there a way to mock something instead of a physical wait?

Reply:
I wasn't able to reliably hit the condition unless I could force the thread to yield while in the critical section. This was the only injection point I could find. A shorter wait would probably do, or I'm happy to hear other suggestions.

Reply:
How do the other import thread tests handle this?

Reply:
Here's a `Finder` that sleeps for 0.01 during `find_spec()`:

https://github.com/python/cpython/blob/7b9d406729e7e7adc482b5b8c920de1874c234d0/Lib/test/test_importlib/test_threaded_import.py#L63-L81